### DISCUSSION TITLE
How to spin the Azure Function in Premium Plan rather than Consumption Plan

### DISCUSSION THREAD

**Discussion by voyager163** [polymind-inc/acmebot#489] [Q&A] [answered]
Hi @shibayan,

Can you advised me how to spin the Azure Function in premium plan rather than consumption plan?

The reason is that i wish to limit the access for the SSL keyvault to only the Azure Function but in consumption plan, there is a limitation.

**Comment by shibayan** [comment, marked as answer]
For cost optimization, only the Consumption Plan template is provided, but it should be available by setting the settings starting with `WEBSITE_RUN_FROM_PACKAGE` and `Acmebot` in App Settings to the Azure Functions created as Premium Plan.
